app/models/knight.rb

```diff
@@ -1,5 +1,14 @@
 class Knight < Piece
   scope :piece, -> {where(type: 'Knight')}
 
+  def valid_move?(new_x, new_y)
+    return false if outside_board?(new_x, new_y)
+    #does not matter if knight is obstructed on its way to its new space; it can jump over other pieces
 
-end
\ No newline at end of file
+    if (self.position_x - new_x).abs + (self.position_y - new_y).abs == 3 &&
```

Comment:
Nice job @drewsef-916 !

we can even skip the `if` and just have this:

```
(self.position_x - new_x).abs + (self.position_y - new_y).abs == 3 &&
(self.position_x - new_x).abs != 0 &&
(self.position_y - new_y).abs != 0
```

It will automatically evaluate to true or false

Reply:
There are some conflicts which need to be resolved though...